calibration-app/gradle.properties

```diff
@@ -33,6 +33,6 @@ android.enableJetifier=true
 # the calibration app directory
 GROUP=org.dpppt
 POM_ARTIFACT_ID=dp3t-sdk-android
-VERSION_CODE=220
-VERSION_NAME=2.2.0
+VERSION_CODE=230
+VERSION_NAME=2.3.0
```

Comment:
That's the wrong file, should be in `/dp3t-sdk/gradle.properties`. Can we add a comment here, that these values are just placeholders, to avoid confusion every time?